data/intakes.yml

```diff
@@ -27,16 +27,16 @@
   course: Full Stack Web Developer Bootcamp
   price: 97 000
 
-- start_date: 3 February 2020 
-  end_date: 24 April 2020
-  precourse_start_date: 6 January 2020
+- start_date: 8 June 2020
+  end_date: 7 September 2020
+  precourse_start_date: 11 May 2020
   locations: "Göteborg, Stockholm, Remote"
-  course: Full Stack Web Developer Bootcamp - NO SEATS AVAILABLE
-  price: 97 000
+  course: Full Stack Web Developer Bootcamp
+  price: 99 500
 
-- start_date: 6 April 2020
-  end_date: 27 June 2020
-  precourse_start_date: 9 March 2020
+- start_date: 24 August 2020
+  end_date: 14 November 2020
+  precourse_start_date: 13 July 2020
```

Comment:
We don't REMOVE dates but rather ADD new intakes. Please reinstate the data that is missing.